iree/test/e2e/models/resnet_fake_weights.mlir

```diff
@@ -0,0 +1,1294 @@
+// ResNet model with placeholder weights, for testing.
+
+// RUN: iree-run-mlir -export-all -iree-hal-target-backends=vmla %s -function-input="1x224x224x3xf32" | IreeFileCheck %s
+// RUN: [[ $IREE_LLVMJIT_DISABLE == 1 ]] || (iree-run-mlir -export-all -iree-hal-target-backends=llvm-ir %s -function-input="1x224x224x3xf32" | IreeFileCheck %s)
+// RUN: [[ $IREE_VULKAN_DISABLE == 1 ]] || (iree-run-mlir -export-all -iree-hal-target-backends=vulkan-spirv %s -function-input="1x224x224x3xf32" | IreeFileCheck %s)
```

Comment:
VMLA+Vulkan takes 16 seconds in `RelWithDebInfo` on my machine. We can remove this and/or LLVMJIT if it ends up being too slow on CI.